resources/leiningen/new/kraken_http_api/PROJECT@.service.template

```diff
@@ -25,12 +21,17 @@ ExecStartPre=/bin/bash -c 'sleep 2 && curl -s http://${COREOS_PRIVATE_IPV4}:8500
 ExecStartPre=/usr/bin/docker pull ${DOCKER_REPO}:${VERSION}
 
 ExecStart=/bin/bash -c 'docker run --name ${CONTAINER} --restart=always \
-  --link rabbitmq:rabbitmq \
-  --link wildfly:wildfly \
+  --dns $COREOS_PRIVATE_IPV4 \
   --env ALLOWED_ORIGINS="$(curl -s http://${COREOS_PRIVATE_IPV4}:8500/v1/kv/{{name}}/allowed-origins?raw)" \
+  --env NEW_RELIC_LICENSE_KEY=$(curl -s http://${COREOS_PRIVATE_IPV4}:8500/v1/kv/new-relic/license-key?raw) \
+  --env RABBITMQ_PORT_5672_TCP_ADDR=$(curl -s http://${COREOS_PRIVATE_IPV4}:8500/v1/kv/rabbitmq/host?raw) \
```

Comment:
Hmm, yeah, now that I think about it, we probably want this in the `-works` template too.